src/sentry/receivers/releases.py

```diff
@@ -71,6 +71,7 @@ def resolved_in_commit(instance, created, **kwargs):
                                 'commit': instance.id,
                             }
                         )
+                    GroupAssignee.objects.assign(group=group, assigned_to=user_list[0])
```

Comment:
What if we make one small change, and say we only create the activity item for the first matching user, which would also be the one we assign it to